examples/oneview_network_set_facts.yml

```diff
@@ -0,0 +1,57 @@
+###
+# Copyright (2016) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+---
+- hosts: all
+  vars:
+    - config: '{{ playbook_dir }}/oneview_config.json'
+    - network_set_name: 'OneViewSDK Test Network Set - Renamed'  # set the name of an existent network set to run this example
```

Comment:
I know that is just an example, but I would have removed 'Renamed' from the name